text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Token API will provide token information by requesting `hathor-wallet-service`.
+
+# Motivation
+[motivation]: #motivation
+
+We need to retrieve token data without requesting the full node. Wallet service holds those data, so, we'll use it for now. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+- We'll basically act as a proxy making requests to wallet service
+
+# Reference-level explanation
```

Comment:
Still in the low level design, how will it be deployed? Lambdas, right? I'm concerned that it can get really expensive if the Wallet Service's Token API hangs for any reason and we keep waiting for them. What would be a reasonable timeout here? Should we deploy it using k8s?